### PR TITLE
Set next assumed release to 2.18.1.

### PR DESCRIPTION
**Pull request by rstam**

### CODE REVIEW

File: Release Notes/Release Notes v2.18.1.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# .NET Driver Version 2.18.1 Release Notes
```
Comment (DmitryLukyanov) [inline Release Notes/Release Notes v2.18.1.md:1]: Are we planning to fill it beforehand? :)

Comment (rstam) [reply Release Notes/Release Notes v2.18.1.md:1]: No... that's why there's a TODO below.

**Review by BorisDog — APPROVED** [review]
LGTM

**Review by DmitryLukyanov — APPROVED** [review]
LGTM

**Pull request merged**